CHANGELOG.md

```diff
@@ -11,53 +11,55 @@ that users understand how the changes affect the new version.
 
 version 2.1.0-dev
 ---------------------------
-+ Updated TALON to version 4.4
++ Updated parameter_meta sections for Minimap2 and TranscriptClean.
++ Updated cores variable for TALON.
```

Comment:
Updated how?

Reply:
Updated to the new wdl-aid structure.

Reply:
I meant the cores variable: `Updated cores variable for TALON.`
Looks like the default value changed, that should be mentioned specifically.

Reply:
Ah oke, I have addressed it in the CHANGELOG now.